src/lesson1/task1/Simple.kt

```diff
@@ -60,31 +64,48 @@ fun seconds(hours: Int, minutes: Int, seconds: Int): Int = TODO()
  * Определить длину того же отрезка в метрах (в данном случае 18.98).
  * 1 сажень = 3 аршина = 48 вершков, 1 вершок = 4.445 см.
  */
-fun lengthInMeters(sagenes: Int, arshins: Int, vershoks: Int): Double = TODO()
+fun lengthInMeters(sagenes: Int, arshins: Int, vershoks: Int): Double {
+    val x = sagenes*48*4.445
+    val y = arshins*16*4.445
+    val z = vershoks*4.445
+    return (x+y+z)/100
+}
 
 /**
  * Тривиальная
  *
  * Пользователь задает угол в градусах, минутах и секундах (например, 36 градусов 14 минут 35 секунд).
  * Вывести значение того же угла в радианах (например, 0.63256).
  */
-fun angleInRadian(grad: Int, min: Int, sec: Int): Double = TODO()
+fun angleInRadian(grad: Int, min: Int, sec: Int): Double {
+    val x = min/60
```

Comment:
Int / Int = Int in Kotlin (and many other languages like C or Java). So e.g. if `min` is 30 your `x` is 30 / 60 = 0. To get Double result here, you should replace 60 with 60.0